src/App.scss

```diff
@@ -1 +0,0 @@
-
```

Comment:
@epwinchell added this file IIRC - not sure if it's a placeholder or if is actually needed to load `scss` from patternfly.

Reply:
Not sure, maybe a placeholder. I stashed the line change on that file.